frontend/pages/hosts/details/cards/Software/SoftwareVulnCount/SoftwareVulnCount.tsx

```diff
@@ -18,14 +19,20 @@ const SoftwareVulnCount = ({
     return software.vulnerabilities?.length ? sum + 1 : sum;
   }, 0);
   return vulnCount ? (
-    <div className={`${baseClass}`}>
+    <InfoBanner className={baseClass} color="grey" borderRadius="xlarge">
       <div className={`${baseClass}__count`}>
         <Icon name="issue" />
         {vulnCount === 1
           ? "1 software item with vulnerabilities detected"
           : `${vulnCount} software items with vulnerabilities detected`}
       </div>
-    </div>
+      {!deviceUser && (
+        <p>
```

Comment:
Saw this on the original Figma but not in our code. @rachaelshaw can you approve this?